config.yml

```diff
@@ -95,6 +95,9 @@ defaults:
 
 
   pulse:
+    # This service, tc-auth, cannot use tc-pulse, because tc-pulse depends on
+    # tc-auth.  So this must be a statically configured pulse user with access
+    # to namepsace 'taskcluster-auth'.
```

Comment:
I guess we don't even need to mention tc-pulse now?

Reply:
Oh, good call..